<launch>
  <!--<node name="trajectory_planner_nodelet" pkg="trajectory_planner" type="trajectory_planner_nodelet" /><node name="traj_executor" pkg="trajectory_planner" type="trajectory_executive" /><node name="APgenerator" pkg="trajectory_planner" type="APgenerator" /><node name="mtt_target_generator" pkg="trajectory_planner" type="mtt_target_generator" />-->
  <node name="trajectory_planner_nodelet" pkg="trajectory_planner" type="trajectory_planner_nodelet" required="true" output="screen">
    <remap from="/mtt_targets" to="/mtt/trck/targets"/>
  </node>
  <node name="traj_executor" pkg="trajectory_planner" type="trajectory_executive" required="true" output="screen" />
  <node name="tracking_frame_generator" pkg="trajectory_planner" type="tracking_frame_generator" required="true" output="screen" />
  <!-- <node name="rviz" pkg="rviz" type="rviz" args="-display-config $(find trajectory_planner)/rvizconfig/rviz.vcg" /> -->
  <node name="rviz" pkg="rviz" type="rviz" />
  <group ns="/pcp/fus/planar_pc">
    <remap from="/pointcloud0" to="/pc_out_pointcloud"/>
    <remap from="/tracking_frame" to="/tracking_frame1"/>
    <remap from="/pc_out" to="/pcp/fus/planar_pc/points"/>
    <node name="planar_pc" pkg="mtt" type="planar_pc_generator" required="true" output="screen">
      <param name="output_frequency" value="10.0"/>
      <param name="perpendicular_treshold" value="0.05"/>
    </node>
  </group>
  <group ns="/pcp/track/mtt">
    <node name="node" pkg="mtt" type="line_fitting" required="true" output="screen">
      <remap from="/points" to="/pcp/fus/planar_pc/points"/>
      <remap from="/markers" to="/pcp/trck/mtt/markers"/>
      <remap from="/marker" to="/pcp/trck/mtt/marker"/>
      <remap from="/targets" to="/mtt/trck/targets"/>
    </node>
  </group>
</launch>  